lib/spack/spack/operating_systems/freebsd.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import platform as py_platform
+
+from spack.version import Version
+
+from ._operating_system import OperatingSystem
+
+
+class FreeBSD_os(OperatingSystem):
```

Comment:
Can we use:
```suggestion
class FreeBSDOs(OperatingSystem):
```
or anything without an underscore in a class name?

Reply:
I found it hard to read, but ok.